docs/CODE_OF_CONDUCT-ja.md

```diff
@@ -0,0 +1,49 @@
+# 貢献者の行動規範
+
+このプロジェクトの貢献者および管理者として、また、開かれた歓迎のコミュニティを育むために
+オープンで歓迎されるコミュニティを育成するために、私たちは、問題の報告、機能追加の要望の投稿、更新を通じて貢献するすべての人々を尊重することを誓います。
+を尊重することを誓います。
+ドキュメントの更新、プルリクエストやパッチの提出、その他の活動を通じて貢献するすべての人々を尊重することを誓います。
+
+私たちは、このプロジェクトへの参加をハラスメントのないものにすることを約束します。
+経験、性別、性自認、性表現、性的指向、障害の有無にかかわらず、すべての人にとってハラスメントのないプロジェクトにすることを約束します。
+アイデンティティと表現、性的指向、身体障害、容姿、体格、人種、民族性
+体格、人種、民族性、年齢、宗教、国籍に関係なく、このプロジェクトに参加するすべての人にハラスメントのない経験を提供することを約束します。
+
+参加者による容認できない行為の例としては、以下が挙げられる：
+
+* 性的な言葉やイメージの使用
+* 個人攻撃
+* 荒らしや侮辱的/中傷的なコメント
+* 公的または私的な嫌がらせ
+* 明示的な許可なく、物理的または電子的な住所など、他人の個人情報を公開すること。
+  明示的な許可なく、他人の住所を公開すること。
+* その他の非倫理的または非プロフェッショナルな行為
+
+プロジェクトメンテナには、以下を削除、編集、拒否する権利と責任があります。
+コメント、コミット、コード、ウィキ編集、課題、その他の貢献を拒否する権利があります。
+この行動規範に沿わない投稿を削除、編集、拒否する権利があります。
+また、不適切と判断されるその他の行為について、投稿者を一時的または恒久的に追放することがあります、
+を一時的または恒久的に禁止することができます。
+
+この行動規範を採用することで、プロジェクトのメンテナーは以下のことを約束します。
+この行動規範を採用することによって、プロジェクトメンテナはこの原則をこのプロジェク
+することを約束するものとします。この行動規範に従わない、あるいは強制しないプロジェクトメンテナは、プロジェクトメンテナから永久に解任されることがあります。
+行動規範に従わないプロジェクトメンテナは、プロジェクトチームから永久に排除される可能性があります。
+
+この行動規範は、プロジェクト空間内および公共の場の両方に適用されます。
+個人がプロジェクトやそのコミュニティを代表しているときに適用されます。
+
+虐待、ハラスメント、その他容認できない行為があった場合は、以下の連絡先まで報告してください。
+プロジェクトメンテナ（victorfelder at gmail.com）までご連絡ください。すべての
+すべての苦情は検討され、調査され、その結果、状況に応じて必要かつ適切とみなされる対応がなされます。
+が必要かつ適切であるとみなされます。メンテナーは
+報告者の秘密を守る義務があります。
+を守らなければならない。
+
```

Comment:
From line 37 to 42, it looks like the same things are explained double.

Reply:
Also made changes to it